packages/api/cms-api/jest.config.ts

```diff
@@ -5,4 +5,5 @@ module.exports = {
     transform: {
         "\\.[jt]sx?$": "ts-jest",
     },
+    testTimeout: 30000,
```

Comment:
Is this still necessary?

Reply:
probably not, i will retest this and maybe we can remove or reduce the timeout.

Reply:
still getting some timeouts locally, i think we should at least raise it to 10000